client/src/signin/Register.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+class Register extends Component {
+  state = {
+    username: "",
+    department: "",
+    password: "",
+    error: ""
+  };
+
+  render() {
+    return (
+      <div class="form-wrapper">
+        <h2>Login</h2>
+        <form onSubmit={this.submitHandler}>
+          <input
+            name="username"
+            id="username"
+            value={this.state.username}
+            placeholder="Username"
+            type="text"
+            required
+            onChange={this.changeHandler}
+          />
+          <input
+            name="department"
+            id="department"
+            value={this.state.department}
+            placeholder="Department"
+            type="text"
+            required
+            onChange={this.changeHandler}
+          />
+          <input
+            name="password"
+            id="password"
+            value={this.state.password}
+            placeholder="Password"
+            type="password"
+            required
+            onChange={this.changeHandler}
+          />
+          <button type="submit">Log In</button>
```

Comment:
Shouldn't this be 'Register'? Which makes me think, this component and the Login component look nearly identical; do you think you could combine them into one component (without giving up any of the current differences)?